### ISSUE TITLE
create local `user` SR Linux user with ssh keys for auth

### ISSUE THREAD

**Issue by hellt** [srl-labs/containerlab#704]
To enable password-less access a local user needs to be created along with a role and rules for that role

example:

```
A:srl1# info / system aaa                                                                                                                         
    authentication {
        user clab {
            ssh-key [
                "ssh-rsa somepubkey"
            ]
            role [
                admin
            ]
        }
    }
    authorization {
        role admin {
        }
    }

A:srl1# info /system configuration role admin                                                                                                     
    configuration {
        role admin {
            rule / {
                action write
            }
        }
    }
```

containerlab needs to read every file with `~/.ssh/*.pub` name signature and populate the found pub keys inside the template we use to post-provision sr linux nodes

**Comment by hellt** [comment]
This role creation is not needed, since we can use `authorized_keys` file that will be read by srlinux -- #716

**Comment by hellt** [comment]
implemented in #716
